index.js

```diff
@@ -0,0 +1,25 @@
+// const dropdown = document.querySelector('.Dropdown__button');
+// const dropdownMenu = document.querySelector('.Dropdown__menu');
+
+// const toggleShow = (event) => {
+//     console.log('clicked');
+//     dropdownMenu.classList.toggle('Dropdown__menu--show');
+// }
+
+// dropdown.addEventListener('click', toggleShow);
+
+//----------left non-class code above so I can pull from for examples later-----------
+
+class Dropdown {
```

Comment:
This dropdown menu is working great. The only thing I can suggest is that when you hover the menu, the mouse doesn't change into a pointer. But this is something minor.

Reply:
I actually did that in the next project :)

Reply:
Oh nice!!